gse-afs-ext-base/src/main/java/com/powsybl/gse/afs/ext/base/ModificationScriptEditor.java

```diff
@@ -160,6 +183,13 @@ public ModificationScriptEditor(StorableScript storableScript, Scene scene, GseC
         storableScript.addListener(this);
     }
 
+    private void addVirtualScript(AbstractScript script, Scene scene, GseContext context) {
+        VirtualScriptCreator virtualScriptCreator = new VirtualScriptCreator(script, scene, context);
+        Callback<ButtonType, Boolean> resultConverter = buttonType -> buttonType == ButtonType.OK ? Boolean.TRUE : Boolean.FALSE;
+        Dialog<Boolean> dialog = new GseDialog<>(virtualScriptCreator.getTitle(), virtualScriptCreator, scene.getWindow(), virtualScriptCreator.okProperty().not(), resultConverter);
+        dialog.showAndWait().filter(result -> result).ifPresent(result -> virtualScriptCreator.create());
```

Comment:
after successful addition of a script (same will be for removal), the included scripts pane should be updated

Reply:
resolved